web/src/main/java/com/futurewei/alcor/web/entity/subnet/SubnetEntity.java

```diff
@@ -36,9 +37,19 @@
     @JsonProperty("availability_zone")
     private String availabilityZone;
 
+    // TODO: considering to put into port
```

Comment:
Yes, I think you've introduced port_detail in the SubnetEntity. 

How about removing the old gw_port_ip and gw_port_mac? I would recommend to keep them for now unless PM and other relevant managers switch to the new contract, and then we remove the old fields. Therefore it might be worth putting a @legacy annotation on those fields.

Reply:
Yes ~ I think so

Reply:
we could do this in future~